client/src/components/ReviewItem.js

```diff
@@ -1,23 +1,60 @@
-import React from 'react'
+import React, { useState } from 'react'
+import EditReviewForm from './EditReviewForm'
 
-const ReviewListItem = ({ title, rating, text, deleteReview, id, currentUser, userId }) => {
+const ReviewListItem = ({ title, rating, text, id, userId, firstName, lastName, currentUser, deleteReview }) => {
+  const [ review, setReview ] = useState({
+    title: title,
+    rating: rating,
+    text: text,
+    firstName: firstName,
+    lastName: lastName
+  })
```

Comment:
we shouldn't need separate state for a single review item, since the parent of this component (such as `ReviewList`) is holding state for _all_ of the reviews as the source of truth, so we should update state there after the review has been editted